generators/csharp/codegen/src/project/CsharpProject.ts

```diff
@@ -504,23 +504,12 @@ class CsProj {
 
 ${projectGroup.join("\n")}
 
-    <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0'">
-        <PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
-    </PropertyGroup>
-
-    <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0'">
-        <PackageReference Include="Portable.System.DateTimeOnly" Version="8.0.1" />
-    </ItemGroup>
-
     <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
         <Reference Include="System.Net.Http" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetFramework)' == 'net7.0' Or '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0'">
-        <PackageReference Include="PolySharp" Version="1.14.1">
-            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-            <PrivateAssets>all</PrivateAssets>
-        </PackageReference>
+    <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Nice, this is way simpler - we don't lose any functionality here?

Reply:
We're always building for all TFMs, so we'll always restore the package anyway.
Technically, the highest TFM doesn't require it so it may improve build speed, but insignificantly.